src/backend/daemon.go

```diff
@@ -1,7 +1,5 @@
-// +build !dockerless
-
 /*
-Copyright 2016 The Kubernetes Authors.
+Copyright 2021 Mirantis
```

Comment:
Probably you had to retain the old copyright header?

https://www.apache.org/licenses/LICENSE-2.0
> 4. Redistribution. ...
> ...
> c. You must retain, in the Source form of any Derivative Works that You distribute, all copyright, patent, trademark, and attribution notices from the Source form of the Work, excluding those notices that do not pertain to any part of the Derivative Works; and
> ...